pkg/storage/export/export/export.go

```diff
@@ -453,8 +453,42 @@ func (ctrl *VMExportController) handleVMI(obj interface{}) {
 			for _, key := range keys {
 				ctrl.vmExportQueue.Add(key)
 			}
+			return
+		}
+		pvcs := ctrl.getPVCsFromVMI(vmi)
+		for _, pvc := range pvcs {
+			ctrl.handlePVC(pvc)
+		}
+	}
+}
+
+func (ctrl *VMExportController) getPVCsFromVMI(vmi *virtv1.VirtualMachineInstance) []*corev1.PersistentVolumeClaim {
+	var pvcs []*corev1.PersistentVolumeClaim
+	for _, volume := range vmi.Spec.Volumes {
+		if volume.PersistentVolumeClaim != nil {
+			if pvc := ctrl.getPVCsFromName(vmi.Namespace, volume.PersistentVolumeClaim.ClaimName); pvc != nil {
+				pvcs = append(pvcs, pvc)
+			}
+		}
+		if volume.DataVolume != nil {
+			if pvc := ctrl.getPVCsFromName(vmi.Namespace, volume.DataVolume.Name); pvc != nil {
+				pvcs = append(pvcs, pvc)
+			}
```

Comment:
What about memDump?  Also, could use `PVCNameFromVirtVolume` helper

Reply:
mem dumps are associated with VMs, not VMIs which is why I didn't include them in this check. I didn't know PVCNameFromVirtVolume existed, let me see if that can work.